src/App.js

```diff
@@ -14,11 +16,87 @@ const goodsFromServer = [
   'Garlic',
 ];
 
-const App = () => (
-  <div className="App">
-    <h1>Selected good: -</h1>
-    {goodsFromServer.length}
-  </div>
-);
+class App extends Component {
+  state = {
+    allItems: goodsFromServer,
+    selectedItems: [],
+  }
+
+  addItem = (selectedItem) => {
+    this.setState((state) => {
+      if (state.selectedItems.includes(selectedItem)) {
+        const newSelectedItems = state.selectedItems.filter(item => (
+          item !== selectedItem
+        ));
+
+        return {
+          selectedItems: newSelectedItems,
+        };
+      }
+
+      const newSelectedItems = [...state.selectedItems, selectedItem];
+
+      return {
+        selectedItems: newSelectedItems,
+      };
+    });
+  }
+
+  removeAllItems = () => {
+    this.setState({
+      selectedItems: [],
+    });
+  }
+
+  render() {
+    const { selectedItems, allItems } = this.state;
+    const { addItem, removeAllItems } = this;
+
+    return (
+      <div className="App">
+        <div className="container">
+          <div className="products">
+            <h2 className="products__title">Add Items to shopping cart</h2>
+            <span
+              className="products__tip"
+            >
```

Comment:
```suggestion
            <span className="products__tip" >
```
переноси если два и больше тегов